pages/signup.js

```diff
@@ -1,9 +1,9 @@
-import SignUpForm from '../components/SignUpForm';
+import SignUpForm from '../containers/SignUpForm';
```

Comment:
@Sly777, thanks for review
I will make as before
Can you explain how you divide between containers and components?